propagation_test.go

```diff
@@ -109,13 +116,12 @@ func TestSpanPropagator(t *testing.T) {
 		assert.Equal(t, exp, sp, formatName)
 	}
 
-	assertMetrics(t, stats, []expectedMetric{
-		{[]string{"jaeger.spans", "group", "sampling", "sampled", "y"}, 1 + 2*len(tests)},
-		{[]string{"jaeger.spans", "group", "lifecycle", "state", "started"}, 1 + 2*len(tests)},
-		{[]string{"jaeger.spans", "group", "lifecycle", "state", "finished"}, 1 + len(tests)},
-		{[]string{"jaeger.traces", "sampled", "y", "state", "started"}, 1},
-		{[]string{"jaeger.traces", "sampled", "y", "state", "joined"}, len(tests)},
-	})
+	counters, _ := stats.Snapshot()
+	assert.EqualValues(t, 1+2*len(tests), counters["jaeger.spans|group=sampling|sampled=y"])
```

Comment:
hm, this looks like a regression... The other API did not force you into knowing how the local backend concatenates the tags into a string.